examples/graphemes_iter.rs

```diff
@@ -28,7 +28,7 @@ impl<'a> RopeGraphemes<'a> {
         let first_chunk = chunks.next().unwrap_or("");
         RopeGraphemes {
             text: *slice,
-            chunks: chunks,
+            chunks,
```

Comment:
I'm not actually a fan of implicit field assignment, especially in contexts like this where most of the fields aren't implicitly assigned.  So I'd prefer to leave this out.